README.md

```diff
@@ -21,6 +21,10 @@ openGemini is a cloud-native distributed time series database, focusing on the s
 
 If you have used openGemini, please let us know and [click submit Usage Information](https://github.com/openGemini/openGemini/issues/62)
 
+![cncf_logo](https://github.com/volcano-sh/volcano/raw/master/docs/images/cncf-logo.png)
```

Comment:
replace cncf_logo from `volcano-sh/volcano` to [https://github.com/cncf](https://github.com/cncf)